data/styles/clear/include/Icons.mapcss

```diff
@@ -691,9 +693,9 @@ node|z16-[tourism=theme_park],
 area|z16-[tourism=theme_park]
 {icon-image: theme_park-m.svg;icon-min-distance: 24;}
 
-node|z17-[tourism=attraction][attraction=animal],
-area|z17-[tourism=attraction][attraction=animal]
-{icon-image: zoo-m.svg;icon-min-distance: 24;}
+node|z17-[attraction=animal],
+area|z17-[attraction=animal]
```

Comment:
nit: trailing comma for consistency

also note that its better to use just "node" line both here and above around line 460
duplicating "node" & "area" lines is redundant in 95% cases